superset/views/core.py

```diff
@@ -1314,6 +1318,8 @@ def testconn(self):
         db_name = request.json.get("name")
         uri = request.json.get("uri")
         try:
+            if app.config.get("PREVENT_UNSAFE_DB_CONNECTIONS"):
```

Comment:
Change `app.config.get("PREVENT_UNSAFE_DB_CONNECTIONS")` -> `app.config["PREVENT_UNSAFE_DB_CONNECTIONS"]` as all configs are required to have defaults.